components/form/input.scss

```diff
@@ -41,8 +26,12 @@ $form-input-border: 1px solid var(--hdd-color-grey);
   }
 
   &.is-disabled {
-    background: var(--hdd-color-grey-light);
-    cursor: default;
+    border-color: var(--hdd-color-grey);;
+    cursor: none;
```

Comment:
Dette er skummelt

Reply:
Endert til not-allowed og flyttet denne korrekt ned til inputfeltet ehr.